### PR TITLE
gh-secret: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #4701

### CODE REVIEW

**Review by bl-ue — COMMENTED** [review]
What do you mean by "local path"?  "Local repo"?

**Comment by navarroaxel** [comment]
> What do you mean by "local path"? "Local repo"?

When your `pwd` is inside a Git repository (has a parent folder with a .git/ dir) the `gh` command assume you want to edit that repository.

**Comment by bl-ue** [comment]
How about changing it to something like "current repo"?  Because that wasn't obvious to me at first, and it may not be (or may be :smile:) to new users.

File: pages/common/gh-secret.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- List secret keys for a specific organization:
+
+`gh secret list --org {{organization}}`
+
+- List secret keys for a specific repository:
+
+`gh secret list --repo {{owner}}/{{repository}}`
+
+- Set a secret from `stdin` for the repository in the local path:
```
Comment (navarroaxel) [inline pages/common/gh-secret.md:18]:
```suggestion
- Set a secret from `stdin` for the current repository:
```

File: pages/common/gh-secret.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# gh secret
+
+> Manage GitHub secrets from the command line.
+> More information: <https://cli.github.com/manual/gh_secret>.
+
+- List secret keys for the repository in the local path:
```
Comment (navarroaxel) [inline pages/common/gh-secret.md:6]:
```suggestion
- List secret keys for the current repository:
```

File: pages/common/gh-secret.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set a secret for a specific repository:
+
+`gh secret set {{name}} --body {{value}} --repo {{owner}}/{{repository}}`
+
+- Set an organization secret for specific repositories:
+
+`gh secret set {{name}} --org {{organization}} --repos {{repository1,repository2}}`
+
+- Remove a secret for the repository in the local path:
```
Comment (navarroaxel) [inline pages/common/gh-secret.md:30]:
```suggestion
- Remove a secret for the current repository:
```

File: pages/common/gh-secret.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- List secret keys for a specific organization:
+
+`gh secret list --org {{organization}}`
+
+- List secret keys for a specific repository:
+
+`gh secret list --repo {{owner}}/{{repository}}`
+
+- Set a secret from `stdin` for the current repository:
```
Comment (bl-ue) [inline pages/common/gh-secret.md:18]:
```suggestion
- Set a secret from a file for the current repository:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
1 comment.  Otherwise LGTM!

**Review by bl-ue — APPROVED** [review]
Nice. It has a full, beautiful 8 examples! 😉

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**